app/hotels/src/navigation/allHotels/SearchResultsScreen.js

```diff
@@ -15,25 +15,31 @@ import { defaultTokens } from '@kiwicom/mobile-orbit';
 import { SafeAreaView } from 'react-navigation';
 
 import NewAllHotels from '../../allHotels/NewAllHotels';
-import NewAllHotelsMap from '../../map/allHotels/NewAllHotelsMap';
```

Comment:
There was the map there before. The designs are somehow unclear about what should happen. 

though I feel it would be weird to show the map there, then when you click on a hotel, there is no way to get back to the map

Reply:
Yes, that's true. Maybe we should check with the designers what their intentions were.
Also, we can fix it later, but it seems like the double column layout is not 50%-50% anymore, more like 35%-65%.